spec/models/document_spec.rb

```diff
@@ -150,11 +150,9 @@
       it { expect { copy_from }.to change(new_document, :verified).to true }
 
       it 'schedules a ConvertDocumentJob (after save)' do
-        old_document # Preload so that the ConvertDocumentJob doesn't affect the test
-        expect do
-          copy_from
-          new_document.save
-        end.to(have_enqueued_job(ConvertDocumentJob).with { |id| expect(id).to eq new_document.reload.to_param })
+        copy_from
+        new_document.save
+        expect(ConvertDocumentJob).to(have_been_enqueued.with(new_document.reload.to_param))
```

Comment:
are the braces necessary?
```suggestion
        expect(ConvertDocumentJob).to have_been_enqueued.with(new_document.reload.to_param)
```